qemu/tests/cfg/smbios.cfg

```diff
@@ -29,6 +29,8 @@
         - type0and1:
             smbios_type = Bios System
         - machine_type:
+            drive_format_image1 = ahci
```

Comment:
`ahci` is for q35 only,  but no this restriction for case `machine_type`. 
for  `i440fx`,  it should be `ide`

Reply:
This case takes each qemu-supported machine type in a loop and checks it in guest. It will meet both pc and q35 in a round. So it's better to choose ahci for compatibility for both pc and q35.

Reply:
Got it, thanks !